compiler/src/iree/compiler/Codegen/Dialect/GPU/Transforms/ExpandUndistributedInnerTiles.cpp

```diff
@@ -0,0 +1,231 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenAttrs.h"
+#include "iree/compiler/Codegen/Dialect/Codegen/IR/IREECodegenOps.h"
+#include "iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUEnums.h"
+#include "iree/compiler/Codegen/Dialect/GPU/Transforms/Passes.h"
+#include "iree/compiler/Codegen/Dialect/GPU/Transforms/Transforms.h"
+#include "mlir/Dialect/Tensor/IR/Tensor.h"
+#include "mlir/Transforms/WalkPatternRewriteDriver.h"
+
+namespace mlir::iree_compiler::IREE::GPU {
+
+#define GEN_PASS_DEF_EXPANDUNDISTRIBUTEDINNERTILESPASS
+#include "iree/compiler/Codegen/Dialect/GPU/Transforms/Passes.h.inc"
+
+namespace {
+struct ExpandUndistributedInnerTilesPass final
+    : impl::ExpandUndistributedInnerTilesPassBase<
+          ExpandUndistributedInnerTilesPass> {
+  using Base::Base;
+  void runOnOperation() override;
+};
+
+LogicalResult materializeOperandExpandedShape(
```

Comment:
If I followed previous discussions properly, some folks don't like large anonymous namespaces and prefer static (and we landed on doing both somehow).
```suggestion
static LogicalResult materializeOperandExpandedShape(
```

Reply:
That probably should be an out-of-namespace static